### PR TITLE
Fix/minimum xlm value

### PR DESCRIPTION
**Pull request by benjaguiloff**
- When user swaps or adds liquidity with xlm token, max balance now considers minimum locked values (base reserve, trustlines and network fee)
- Also added message when hovering over "available balance" so user knows about this.

### CODE REVIEW

File: src/components/CurrencyInputPanel/CurrencyBalance.tsx [hunk 1]
```diff
@@ -39,20 +42,58 @@ export default function CurrencyBalance({
...
   const theme = useTheme();
+
   return (
     <RowFixed style={{ height: '17px' }}>
-      <BodySmall color={theme.palette.secondary.main}>
-        {!hideBalance && currency ? `Balance: ${balance}` : null}
-      </BodySmall>
-      {showMaxButton ? (
-        <StyledBalanceMax onClick={() => onMax(balance as string)}>Max</StyledBalanceMax>
+      {currency.code === 'XLM' ? (
```
Comment (joaquinsoza) [inline src/components/CurrencyInputPanel/CurrencyBalance.tsx:69]: Should use the XLM contract and not CODE (there could be another asset with the same code)

should use xlmTokenList.find()

File: src/components/CurrencyInputPanel/CurrencyBalance.tsx [hunk 2]
```diff
@@ -39,20 +42,58 @@ export default function CurrencyBalance({
...
 }: CurrencyBalanceProps) {
+  // const [fee, setFee] = useState<number>(0);
+  // const [adjustedBalance, setAdjustedBalance] = useState<number>(0);
   const { tokenBalancesResponse } = useGetMyBalances();
   const balance =
     tokenBalancesResponse?.balances?.find((b) => b?.contract === currency?.contract)?.balance ||
     '0';
 
+  let availableBalance: number;
+  if (currency.code === 'XLM') {
```
Comment (joaquinsoza) [inline src/components/CurrencyInputPanel/CurrencyBalance.tsx:56]: Should use the XLM contract and not CODE (there could be another asset with the same code)

should use xlmTokenList.find()

**Comment by esteblock** [comment]
@MattPoblete https://github.com/soroswap/frontend/issues/304

**Comment by chopan123** [comment]
> * Dapp waits you to put an amount in order to simulate the tx and know the txfees.
> 
> * then when you press MAX, it calcualtes again and so on, infinite loop

moved to issues #359 and #358

**Comment by chopan123** [comment]
New PR on #360

**Pull request closed without merging**